linux_os/guide/services/mail/package_mailx_installed/rule.yml

```diff
@@ -12,12 +14,18 @@ rationale: |-
 
 severity: medium
 
+identifiers:
+    cce@sle12: CCE-92331-8
+    cce@sle15: CCE-92519-8
+
 references:
     disa: CCI-001744
     nist: CM-3(5)
-    srg: SRG-OS-000363-GPOS-00150
```

Comment:
The `SRG-OS-000363-GPOS-00150` is about notification and I think it makes sense in this rule. But `SRG-OS-000445-GPOS-00199` and `SRG-OS-000446-GPOS-00200` are more about verification. I can imagine the `mailx` is somehow related to verification since it can notify finds, but I am not so sure if its precise to include these SRGs here. Could you share your conclusion so I can better understand the context, please?

Reply:
Thanks, @marcusburghardt 

Because it was some time ago, now when I am looking at it I cannot remember what were my thoughts about this change, even more, sounds more logical to be only SRG-OS-000363-GPOS-00150
hmm, it could be some kind of typo on my end, anyway I've reverted this change, and if I have some reason in the future to make corrections here I will add them in a new PR and will explain.

Best regards and have a nice day